src/main/fc/rc_modes.c

```diff
@@ -41,7 +42,8 @@
 
 boxBitmask_t rcModeActivationMask; // one bit per mode defined in boxId_e
 
-static bool paralyzeModeEverDisabled = false;
+static boxBitmask_t STICKY_MODES = {{0, 256}}; // Paralyze
+static boxBitmask_t stickyModesEverDisabled = {{0, 0}};
```

Comment:
I think this is best left to the default initialiser - this way it's not tied to the bitmask having a fixed size.

Reply:
@mikeller fixed